acinclude.m4

```diff
@@ -218,7 +218,7 @@ AC_ARG_ENABLE(qt,
 if test $USE_QT = yes ; then
     PKG_CHECK_MODULES([QTCORE], [Qt5Core >= 5.2])
     PKG_CHECK_VAR([QTBINPATH], [Qt5Core >= 5.2], [host_bins])
-    PKG_CHECK_MODULES([QT], [Qt5Core Qt5Gui Qt5Widgets >= 5.2])
```

Comment:
This breaks compilation with autoconf on non-X11 systems.

Please split it into it's own dependency as we do with QtMultimedia for the qtaudio plugin.

Reply:
You're right. Reworked it.